automatic/_dmd/tools/ChocolateyInstall.ps1

```diff
@@ -23,3 +23,7 @@ Install-ChocolateyZipPackage @packageArgs
 
 $Bin = Join-Path -Resolve $destDir "dmd2/windows/bin"
 $Bin64 = Join-Path -Resolve $destDir "dmd2/windows/bin64"
+
+
+# Add dmd to PATH
+Install-ChocolateyPath -PathToInstall $Bin -PathType User

```

Comment:
Should probably test whether `$Bin` or `$Bin64` make more sense. From a quick google search, I found this that may be helpful: https://docs.chocolatey.org/en-us/create/functions/get-osarchitecturewidth

Reply:
I ignored this because dmd is not publisjed